tests/unit_tests/pal_storage_kv/testcase.yaml

```diff
@@ -1,7 +1,19 @@
 tests:
-  sidewalk.unit_tests.storage_kv:
+  sidewalk.unit_tests.storage_kv.NVS:
     sysbuild: true
     platform_allow: native_posix
     tags: Sidewalk
     integration_platforms:
       - native_posix
+
+  sidewalk.unit_tests.storage_kv.ZMS:
```

Comment:
Do we need such testcase in mainline?

Reply:
Well now we will use two backends for settings, and alto this test runs on native_posix, I think that we should not exclude one of the used backends, simply because it is newer.

Maybe  we should consider if this test is useful at all. Perhaps we should drop it, and rely only on the one run on real hardware.